src/scripts/main.js

```diff
@@ -355,6 +355,25 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+// console.log(people);
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+people.forEach((person) => {
+  const { names, sex, born, died } = person;
+  const age = died - born;
+  const century = Math.ceil(died / 100);
+  const gender = sex === 'f' ? 'female' : 'male';
+
+  table.firstElementChild.insertAdjacentHTML(
```

Comment:
It's a code quality issue. You are directly manipulating the DOM in a loop. This is inefficient because each manipulation causes the browser to recalculate layout and repaint the screen. A better approach would be to create a string of all the HTML you want to add, and then add that string to the innerHTML of the parent element after the loop.